doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -476,106 +421,172 @@ Induction
-   variables that are inside some constructors in the induction hypothesis. The
-   other ones need not be further generalized.
 
-.. tacv:: dependent destruction @ident
+.. tacn:: fix @ident @natural {? with {+ ( @ident {* @simple_binder } {? %{ struct @name %} } : @type ) } }
 
-   This performs the generalization of the instance :n:`@ident` but uses
-   ``destruct`` instead of induction on the generalized hypothesis. This gives
-   results equivalent to ``inversion`` or ``dependent inversion`` if the
-   hypothesis is dependent.
+   .. insertprodn simple_binder simple_binder
 
-See also the larger example of :tacn:`dependent induction`
-and an explanation of the underlying technique.
+   .. prodn::
+      simple_binder ::= @name
+      | ( {+ @name } : @term )
 
-.. seealso:: :tacn:`functional induction`
+   A primitive tactic to start a proof by induction. Generally,
+   higher-level tactics such as :tacn:`induction` or :tacn:`elim`
+   are easier to use.
 
-.. tacn:: discriminate @term
-   :name: discriminate
+   :n:`@ident` is the name of
+   the induction hypothesis. :n:`@natural` tells on which
+   premise of the current goal the induction acts, starting from 1,
+   counting both dependent and non-dependent products, but skipping local
+   definitions. The current lemma must be composed of at
+   least :n:`@natural` products.
 
-   This tactic proves any goal from an assumption stating that two
-   structurally different :n:`@term`\s of an inductive set are equal. For
-   example, from :g:`(S (S O))=(S O)` we can derive by absurdity any
-   proposition.
+   As in a fix expression, induction hypotheses must be used on
+   structurally smaller arguments. The verification that inductive proof
+   arguments are correct is done only when registering the
+   lemma in the global environment. To know if the use of induction hypotheses
+   is correct during the interactive development of a proof, use
+   the command :cmd:`Guarded` (see Section :ref:`requestinginformation`).
 
-   The argument :n:`@term` is assumed to be a proof of a statement of
-   conclusion :n:`@term = @term` with the two terms being elements of an
-   inductive set. To build the proof, the tactic traverses the normal forms
-   [1]_ of the terms looking for a couple of subterms :g:`u` and :g:`w` (:g:`u`
-   subterm of the normal form of :n:`@term` and :g:`w` subterm of the normal
-   form of :n:`@term`), placed at the same positions and whose head symbols are
-   two different constructors. If such a couple of subterms exists, then the
-   proof of the current goal is completed, otherwise the tactic fails.
+   :n:`with {+ ( @ident {* @simple_binder } {? %{ struct @name %} } : @type ) }`
+     Starts a proof by mutual induction. The statements to be proven
+     are :n:`forall @simple_binder__i, @type__i`.
+     The identifiers :n:`@ident` are the names of the induction hypotheses. The identifiers
+     :n:`@ident` are the respective names of the premises on which the induction
```

Comment:
```suggestion
     :n:`@name` are the respective names of the premises on which the induction
```

(there used to be prime symbols to distinguish the two sets of `ident` that were lost during the Sphinx translation, see https://coq.inria.fr/distrib/V8.7.2/refman/tactics.html#hevea_tactic127).

Reply:
OK?